.github/workflows/ci.yml

```diff
@@ -655,7 +655,7 @@ jobs:
     timeout-minutes: 60
     needs:
       - determine-jobs
-    if: needs.determine-jobs.outputs.docker == 'true'
+    if: github.event_name == 'push' && needs.determine-jobs.outputs.docker == 'true'
```

Comment:
I'm confused: isn't this going to _restrict_ the docker job to the sole `main` branch? Is that the intent?

Reply:
Argh - should have been `||`